addon/components/mdc-layout-grid.js

```diff
@@ -0,0 +1,33 @@
+import Ember from 'ember';
+import layout from '../templates/components/mdc-layout-grid';
+
+const { computed } = Ember;
+
+export default Ember.Component.extend({
+  //region Ember Hooks
+  classNames: ['mdc-layout-grid'],
+  classNameBindings: [
+    'fixed-column-width:mdc-layout-grid--fixed-column-width',
+    'isAlignLeft:mdc-layout-grid--align-left',
+    'isAlignRight:mdc-layout-grid--align-right'
+  ],
+  layout,
+  //endregion
+
+  //region Attributes
+  /**
+   * @type {Boolean}
+   */
+  'fixed-column-width': false,
```

Comment:
If possible can you please list the `Attributes` before the `Ember Hooks`? It helps people who dive into the code to find the "configurable" parts more easily.

Reply:
Done!